templates/tools/dockerfile/interoptest/grpc_interop_ruby/Dockerfile.template

```diff
@@ -14,11 +14,12 @@
   # See the License for the specific language governing permissions and
   # limitations under the License.
   
-  <%include file="../../debian_jessie_header.include"/>
+  FROM debian:buster
   
   <%include file="../../apt_get_basic.include"/>
   <%include file="../../python_deps.include"/>
   <%include file="../../ruby_deps.include"/>
+  <%include file="../../cmake.include"/>
```

Comment:
why do we need to include cmake dependencies in this docker image?

Note that this docker image is used for interop tests only, and the ruby test that's changed here (which now builds protoc with cmake), doesn't run in this docker image

Reply:
removed cmake from ruby interop docker image.